.rubocop.yml

```diff
@@ -5,3 +5,6 @@ AllCops:
 
 Metrics/MethodLength:
   Max: 15
+
+Style/Documentation:
```

Comment:
I'm trying to go with the default Rubocops. I've been trying it out and so far it wasn't too bad to keep this documentation cop enabled.

Thoughts?

Reply:
I will remove these this change.